ceph-ansible-pr-syntax-check/build/build

```diff
@@ -8,7 +8,7 @@ set -e
 # which would enforce latest ansible version to be installed.
 virtualenv $TEMPVENV
 "$VENV"/pip install --upgrade pip
-"$VENV"/pip install -r <(grep ansible "$WORKSPACE"/ceph-ansible/tests/requirements.txt)
+"$VENV"/pip install -r "$WORKSPACE"/ceph-ansible/tests/requirements.txt
```

Comment:
In fact, I don't really know why we are installing the tests/requirements.txt there when we can only install the basic requirements (which includes netaddr) [1]

Could you do that change ?

[1] https://github.com/ceph/ceph-ansible/blob/master/requirements.txt

Reply:
Done.